docs/src/components/home/sections/HeroSection.tsx

```diff
@@ -0,0 +1,141 @@
+import NextLink from 'next/link';
+import {
+  Button,
+  Flex,
+  Heading,
+  Icon,
+  Text,
+  useBreakpointValue,
+  View,
+} from '@aws-amplify/ui-react';
+import { MdChevronRight, MdArrowDownward } from 'react-icons/md';
+import {
+  RiCloudWindyLine,
+  RiDragDropLine,
+  RiPencilRuler2Line,
+} from 'react-icons/ri';
+
+import { useCustomRouter } from '@/components/useCustomRouter';
+import { HomeFeatureCard } from '../HomeFeatureCard';
+import { FRAMEWORKS } from '@/data/frameworks';
+import { FrameworkLogo } from '@/components/Logo';
+import { TerminalCommand } from '@/components/InstallScripts';
+
+export const HeroSection = () => {
+  const {
+    query: { platform = 'react' },
+  } = useCustomRouter();
+
+  const showEditor = useBreakpointValue({
+    base: false,
+    large: true,
+  });
+
+  const installScripts = {
+    react: `npm i @aws-amplify/ui-${platform} aws-amplify`,
+    vue: `npm i @aws-amplify/ui-${platform} aws-amplify`,
+    angular: `npm i @aws-amplify/ui-${platform} aws-amplify`,
+    flutter: 'flutter pub add amplify_authenticator',
+  };
+  const frameworkInstallScript = installScripts[platform.toString()];
+
+  return (
+    <View as="section" className="docs-hero">
+      <Flex justifyContent="center" textAlign="center">
+        <Flex
+          direction="column"
+          className="docs-home-subsection--thin"
+          alignItems="center"
+        >
+          <Heading level={1} marginBlockEnd="0">
+            Don't just prototype. <br />
+            <strong>Connect your UI to the cloud.</strong>
+          </Heading>
+          <Text
+            fontSize={{
+              base: 'large',
+              small: 'xl',
+            }}
+          >
+            Amplify UI is a collection of accessible, themeable, performant
+            React{' '}
+            <View as="span" display="inline-block">
+              (and more!
```

Comment:
IMO, it looks a bit cluttered to include all the framework icons in parentheses in the initial text description of what Amplify UI is. I think this would look more concise and impactful:

"Amplify UI is a collection of accessible, themeable, performant React components that can connect directly to the cloud."

We could mention in the Authenticator section that it is available in other frameworks besides React. Thoughts?

Reply:
I agree, let me think about it a bit more. The one issue with that is the framework logos change the homepage to `/angular` or `/vue` which changes the content of the page so it might be tricky.